images/temporal-server/README.md

```diff
@@ -0,0 +1,51 @@
+<!--monopod:start-->
+# temporal-server
+| | |
+| - | - |
+| **OCI Reference** | `cgr.dev/chainguard/temporal-server` |
+
+
+* [View Image in Chainguard Academy](https://edu.chainguard.dev/chainguard/chainguard-images/reference/temporal-server/overview/)
+* [View Image Catalog](https://console.enforce.dev/images/catalog) for a full list of available tags.
+* [Contact Chainguard](https://www.chainguard.dev/chainguard-images) for enterprise support, SLAs, and access to older tags.*
+
+---
+<!--monopod:end-->
+
+Minimal Temporal-Server Image
+
+## Get It!
+
+The image is available on `cgr.dev`:
+
+```
+docker pull cgr.dev/chainguard/temporal-server:latest
+```
+
+## Usage
+
```

Comment:
Can we perhaps add some links to the upstream image / documentation, and also a brief summary of what the image is for? 

Just so if someone lands on this image they know generally what it is, and where it came from

Reply:
For sure, thanks!